### DISCUSSION TITLE
[Backend] Scala 3 with tapir and ZIO

### DISCUSSION THREAD

**Discussion by tpacuszkapl** [realworld-apps/realworld#1124] [WIP Implementations]
Hi, I'm starting an implementation of real world in:
https://www.scala-lang.org/
https://github.com/softwaremill/tapir
https://zio.dev/

Link: https://github.com/softwaremill/realworld-tapir-zio

**Comment by geromegrignon** [comment]
Awesome @tpacuszkapl 🚀
